scripts/pipeline-launch/launch.sh

```diff
@@ -1052,6 +1052,36 @@ echo
 ######################################################
 
 
+######################################################
+echo "Create package manager restriction wrappers"
+echo "-"
+######################################################
+
+CP_USR_BIN="/usr/cpbin"
+
+mkdir -p "$CP_USR_BIN"
+IFS=',' read -r -a RESTRICTING_PACKAGE_MANAGERS <<< "$CP_RESTRICTING_PACKAGE_MANAGERS"
+
+for MANAGER in "${RESTRICTING_PACKAGE_MANAGERS[@]}"
+do
+    MANAGER_PATH=$(which "$MANAGER")
```

Comment:
Are we 100% sure that which will be available at this moment?

Reply:
The usage of `which` was replaced with the usage of `command -v` which is a more common approach. Plain `ubuntu:16.04`, `ubuntu:18.04` and `centos:7` images was tested to have `command` available.

Done in 44f08c878f1637b5b845416f720565dbf31add13.